generator/template.go

```diff
@@ -581,6 +581,8 @@ func mapWellKnownType(protoType string) string {
 		return "StructPBValue[]"
 	case ".google.protobuf.Struct":
 		return "{ [key: string]: StructPBValue }"
+	case ".google.protobuf.FieldMask":
```

Comment:
I see [here](https://googleapis.dev/nodejs/vision/latest/google.protobuf.IFieldMask.html) that `FieldMask` type is `Array.<string> \| null `so should this return `"string[] | null"`?

Reply:
The [docs clarify](https://protobuf.dev/reference/protobuf/google.protobuf/#json-encoding-field-masks) that for JSON encoding (which is what the client receives and sends), there's a special case where it's converted to and from a single string with comma-separated field names. So that's what this type is targeting.